### PR TITLE
Docs: expanding the `freezeColumn()` API ref (#8536)

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR expands the API ref of the `freezeColumn()` method (part of the `ManualColumnFreeze` plugin.

[skip changelog]

### CODE REVIEW

File: docs/next/guides/building-and-testing/building.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ During this process, the source files located in the `src/*` directory are trans
 * `./dist/handsontable.full.min.css`
 * `./dist/languages/*`
 
-More info about dist packages can be found [here](https://github.com/handsontable/handsontable/blob/master/dist/README.md.
+More info about dist packages can be found [here](https://github.com/handsontable/handsontable/blob/master/dist/README.md).
```
Comment (swistach) [inline docs/next/guides/building-and-testing/building.md:45]: Can we move this change to the separated PR (as grouped "documentation polishing" changes)?

File: src/plugins/manualColumnFreeze/manualColumnFreeze.js [hunk 2]
```diff
@@ -85,7 +85,10 @@ export class ManualColumnFreeze extends BasePlugin {
   }
 
   /**
-   * Freezes the given column (add it to fixed columns).
+   * Freezes the specified column (i.e. adds it to fixed columns).
```
Comment (swistach) [inline src/plugins/manualColumnFreeze/manualColumnFreeze.js:88]: I suppose it's a false-positive error. Maybe there is a way to configure the `eslint-*` package to accept `i.e. .... lower case` syntax 🙈

**Review by swistach — CHANGES_REQUESTED** [review]

File: src/plugins/manualColumnFreeze/manualColumnFreeze.js (hunk 2 above)
Comment (kirszenbaum) [reply src/plugins/manualColumnFreeze/manualColumnFreeze.js:88]: @swistach - should I rephrase it? (we use `i.e.` in other places in docs as well)

**Review by AMBudnik — DISMISSED** [review]
Thanks for adding the info about the `render()` @kirszenbaum

**Review by AMBudnik — APPROVED** [review]

**Review by swistach — APPROVED** [review]
:+1:

**Pull request merged**